cmd/helpMessages.go

```diff
@@ -172,6 +172,14 @@ Log in using a VM's user-assigned identity with an Object ID of the service iden
 Log in using a VM's user-assigned identity with a Resource ID of the service identity:
 - azcopy login --identity --identity-resource-id "/subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID"
 
+Log in as a service principal using a client secret:
```

Comment:
Consider adding a notice that users should be mindful of exposing their client secret in the shell history.